flink-runtime/src/test/java/org/apache/flink/runtime/operators/coordination/RecreateOnResetOperatorCoordinatorTest.java

```diff
@@ -264,6 +264,19 @@ public void testConsecutiveResetToCheckpoint() throws Exception {
                 "Timed out when waiting for the coordinator to close.");
     }
 
+    @Test
+    public void testNotifyCheckpointAbortedSuccess() throws Exception {
+        TestingCoordinatorProvider provider = new TestingCoordinatorProvider(null);
+        MockOperatorCoordinatorContext context =
+                new MockOperatorCoordinatorContext(OPERATOR_ID, NUM_SUBTASKS);
+        RecreateOnResetOperatorCoordinator coordinator = createCoordinator(provider, context);
+        TestingOperatorCoordinator internalCoordinatorAfterReset =
+                getInternalCoordinator(coordinator);
+
+        coordinator.notifyCheckpointAborted(1L);
+        assertThat(internalCoordinatorAfterReset.getLastCheckpointAborted()).isEqualTo(1L);
```

Comment:
@liming30 , thanks for the update,  I left a minor comment.

nits:

```suggestion
        long checkpointId = 10L;
        coordinator.notifyCheckpointAborted(checkpointId);
        assertThat(internalCoordinatorAfterReset.getLastCheckpointAborted()).isEqualTo(checkpointId);
```

Reply:
@1996fanrui thanks, I have updated this part of the code.